docs/docs/reference/handlers/postgres.md

```diff
@@ -7,26 +7,138 @@ permalink: docs/reference/handlers/postgres.html
 ---
 
 ## PostgreSQL
-The PostgreSQL handler authenticates incoming connections for a particular
-listener.
 
-**Handler in alpha - not suitable for production** 
+**WARNING: Handler in alpha - not suitable for production** 
```

Comment:
it's worth discussing when we want to remove this warning - if the default is SSL is required, is it safe to remove?